<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="DheenatalansAir.fios-router.home" failures="6" tests="47" name="Default test" time="85.19" errors="0" timestamp="2024-09-28T23:11:34 EDT">
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="6.196"/>
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="2.282"/>
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="0.589"/>
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="0.955"/>
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="0.746"/>
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="1.693"/>
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="1.436"/>
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="0.674"/>
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="0.544"/>
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="1.227"/>
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="0.673"/>
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="0.571"/>
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="0.572"/>
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="25.502">
    <failure type="java.lang.AssertionError" message="The following asserts failed:
	Failure in scenario: valid all | Expected status code: 201 but got: 409 | Error Status: 409 CONFLICT | Error Message: User already exist with same email id,
	Failure in scenario: valid mandatory | Expected status code: 201 but got: 400 | Error Status: 400 BAD_REQUEST | Error Message: Plot number should contain alphaNumeric values only,
	Failure in scenario: firstname as empty | Expected status code: 400 but got: 409 | Error Status: 409 CONFLICT | Error Message: User already exist with same email id,
	Failure in scenario: firstname as numbers | Expected status code: 400 but got: 409 | Error Status: 409 CONFLICT | Error Message: User already exist with same email id,
	Failure in scenario: firstname as special | Expected status code: 400 but got: 409 | Error Status: 409 CONFLICT | Error Message: User already exist with same email id,
	Failure in scenario: lastname as empty | Expected status code: 400 but got: 409 | Error Status: 409 CONFLICT | Error Message: User already exist with same email id,
	Failure in scenario: lastname as numbers | Expected status code: 400 but got: 409 | Error Status: 409 CONFLICT | Error Message: User already exist with same email id,
	Failure in scenario: lastname as special | Expected status code: 400 but got: 409 | Error Status: 409 CONFLICT | Error Message: User already exist with same email id,
	Failure in scenario: phoneNumber as empty | Expected status code: 400 but got: 409 | Error Status: 409 CONFLICT | Error Message: User already exist with same email id,
	Failure in scenario: phoneNumber as less than 10 | Expected status code: 400 but got: 409 | Error Status: 409 CONFLICT | Error Message: User already exist with same email id,
	Failure in scenario: email as empty | Expected status code: 400 but got: 409 | Error Status: 409 CONFLICT | Error Message: User already exist with same contact number,
	Failure in scenario: email as number | Expected status code: 400 but got: 409 | Error Status: 409 CONFLICT | Error Message: User already exist with same contact number,
	Failure in scenario: email as begin with special | Expected status code: 400 but got: 409 | Error Status: 409 CONFLICT | Error Message: User already exist with same contact number,
	Failure in scenario: email as end with special | Expected status code: 400 but got: 409 | Error Status: 409 CONFLICT | Error Message: User already exist with same contact number,
	Failure in scenario: email as end with period | Expected status code: 400 but got: 409 | Error Status: 409 CONFLICT | Error Message: User already exist with same contact number,
	Failure in scenario: plotnumber as empty | Expected status code: 400 but got: 409 | Error Status: 409 CONFLICT | Error Message: User already exist with same email id,
	Failure in scenario: plotnumber as number | Expected status code: 400 but got: 409 | Error Status: 409 CONFLICT | Error Message: User already exist with same email id,
	Failure in scenario: plotnumber as string | Expected status code: 400 but got: 409 | Error Status: 409 CONFLICT | Error Message: User already exist with same email id,
	Failure in scenario: street as empty | Expected status code: 400 but got: 409 | Error Status: 409 CONFLICT | Error Message: User already exist with same email id,
	Failure in scenario: street as number | Expected status code: 400 but got: 409 | Error Status: 409 CONFLICT | Error Message: User already exist with same email id,
	Failure in scenario: street as special | Expected status code: 400 but got: 409 | Error Status: 409 CONFLICT | Error Message: User already exist with same email id,
	Failure in scenario: state as empty | Expected status code: 400 but got: 409 | Error Status: 409 CONFLICT | Error Message: User already exist with same email id,
	Failure in scenario: state as number | Expected status code: 400 but got: 409 | Error Status: 409 CONFLICT | Error Message: User already exist with same email id,
	Failure in scenario: state as special | Expected status code: 400 but got: 409 | Error Status: 409 CONFLICT | Error Message: User already exist with same email id,
	Failure in scenario: country as empty | Expected status code: 400 but got: 409 | Error Status: 409 CONFLICT | Error Message: User already exist with same email id,
	Failure in scenario: country as number | Expected status code: 400 but got: 409 | Error Status: 409 CONFLICT | Error Message: User already exist with same email id,
	Failure in scenario: country as special | Expected status code: 400 but got: 409 | Error Status: 409 CONFLICT | Error Message: User already exist with same email id,
	Failure in scenario: zipcode as empty | Expected status code: 400 but got: 409 | Error Status: 409 CONFLICT | Error Message: User already exist with same email id,
	Failure in scenario: contentType | Expected status code: 415 but got: 409 | Error Status: 409 CONFLICT | Error Message: User already exist with same email id">
      <![CDATA[java.lang.AssertionError: The following asserts failed:
Failure in scenario: valid all | Expected status code: 201 but got: 409 | Error Status: 409 CONFLICT | Error Message: User already exist with same email id,
Failure in scenario: valid mandatory | Expected status code: 201 but got: 400 | Error Status: 400 BAD_REQUEST | Error Message: Plot number should contain alphaNumeric values only,
Failure in scenario: firstname as empty | Expected status code: 400 but got: 409 | Error Status: 409 CONFLICT | Error Message: User already exist with same email id,
Failure in scenario: firstname as numbers | Expected status code: 400 but got: 409 | Error Status: 409 CONFLICT | Error Message: User already exist with same email id,
Failure in scenario: firstname as special | Expected status code: 400 but got: 409 | Error Status: 409 CONFLICT | Error Message: User already exist with same email id,
Failure in scenario: lastname as empty | Expected status code: 400 but got: 409 | Error Status: 409 CONFLICT | Error Message: User already exist with same email id,
Failure in scenario: lastname as numbers | Expected status code: 400 but got: 409 | Error Status: 409 CONFLICT | Error Message: User already exist with same email id,
Failure in scenario: lastname as special | Expected status code: 400 but got: 409 | Error Status: 409 CONFLICT | Error Message: User already exist with same email id,
Failure in scenario: phoneNumber as empty | Expected status code: 400 but got: 409 | Error Status: 409 CONFLICT | Error Message: User already exist with same email id,
Failure in scenario: phoneNumber as less than 10 | Expected status code: 400 but got: 409 | Error Status: 409 CONFLICT | Error Message: User already exist with same email id,
Failure in scenario: email as empty | Expected status code: 400 but got: 409 | Error Status: 409 CONFLICT | Error Message: User already exist with same contact number,
Failure in scenario: email as number | Expected status code: 400 but got: 409 | Error Status: 409 CONFLICT | Error Message: User already exist with same contact number,
Failure in scenario: email as begin with special | Expected status code: 400 but got: 409 | Error Status: 409 CONFLICT | Error Message: User already exist with same contact number,
Failure in scenario: email as end with special | Expected status code: 400 but got: 409 | Error Status: 409 CONFLICT | Error Message: User already exist with same contact number,
Failure in scenario: email as end with period | Expected status code: 400 but got: 409 | Error Status: 409 CONFLICT | Error Message: User already exist with same contact number,
Failure in scenario: plotnumber as empty | Expected status code: 400 but got: 409 | Error Status: 409 CONFLICT | Error Message: User already exist with same email id,
Failure in scenario: plotnumber as number | Expected status code: 400 but got: 409 | Error Status: 409 CONFLICT | Error Message: User already exist with same email id,
Failure in scenario: plotnumber as string | Expected status code: 400 but got: 409 | Error Status: 409 CONFLICT | Error Message: User already exist with same email id,
Failure in scenario: street as empty | Expected status code: 400 but got: 409 | Error Status: 409 CONFLICT | Error Message: User already exist with same email id,
Failure in scenario: street as number | Expected status code: 400 but got: 409 | Error Status: 409 CONFLICT | Error Message: User already exist with same email id,
Failure in scenario: street as special | Expected status code: 400 but got: 409 | Error Status: 409 CONFLICT | Error Message: User already exist with same email id,
Failure in scenario: state as empty | Expected status code: 400 but got: 409 | Error Status: 409 CONFLICT | Error Message: User already exist with same email id,
Failure in scenario: state as number | Expected status code: 400 but got: 409 | Error Status: 409 CONFLICT | Error Message: User already exist with same email id,
Failure in scenario: state as special | Expected status code: 400 but got: 409 | Error Status: 409 CONFLICT | Error Message: User already exist with same email id,
Failure in scenario: country as empty | Expected status code: 400 but got: 409 | Error Status: 409 CONFLICT | Error Message: User already exist with same email id,
Failure in scenario: country as number | Expected status code: 400 but got: 409 | Error Status: 409 CONFLICT | Error Message: User already exist with same email id,
Failure in scenario: country as special | Expected status code: 400 but got: 409 | Error Status: 409 CONFLICT | Error Message: User already exist with same email id,
Failure in scenario: zipcode as empty | Expected status code: 400 but got: 409 | Error Status: 409 CONFLICT | Error Message: User already exist with same email id,
Failure in scenario: contentType | Expected status code: 415 but got: 409 | Error Status: 409 CONFLICT | Error Message: User already exist with same email id
at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:46)
at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:30)
at stepdefinition.createUsertest.tearDown(createUsertest.java:231)
]]>
    </failure>
  </testcase> <!-- runScenario -->
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="0.68"/>
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="0.591"/>
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="0.523"/>
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="0.571">
    <failure type="java.lang.AssertionError" message="The following asserts failed:
	Failure in scenario: empty userid | Expected status code: 400 but got: 404">
      <![CDATA[java.lang.AssertionError: The following asserts failed:
Failure in scenario: empty userid | Expected status code: 400 but got: 404
at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:46)
at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:30)
at stepdefinition.getUserByIDtest.validate_user_and_other_validations_for_scenario_name_as(getUserByIDtest.java:165)
at ✽.Validate "400" and other user validations for scenario name as "empty userid"(file:///Users/mathumathibalakrishnan/git/userAPI_RestAssured/userAPI_RestAssured/src/test/resources/features/04_getUserByID.feature:17)
]]>
    </failure>
  </testcase> <!-- runScenario -->
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="0.198"/>
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="0.522"/>
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="0.588"/>
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="0.537"/>
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="0.536"/>
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="0.557"/>
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="0.529"/>
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="23.595">
    <failure type="java.lang.AssertionError" message="The following asserts failed:
	Failure in scenario: valid mandatory | Expected status code: 200 but got: 400 | Error Status: 400 BAD_REQUEST | Error Message: Plot number should contain alphaNumeric values only,
	Failure in scenario: invalid endpoint | Expected status code: 404 but got: 400 | Error Status: 400 BAD_REQUEST | Error Message: Contact number is already in use by another user,
	Failure in scenario: null endpoint | Expected status code: 404 but got: 400 | Error Status: 400 BAD_REQUEST | Error Message: Contact number is already in use by another user,
	Failure in scenario: phoneNumber as duplicate | Expected status code: 409 but got: 400 | Error Status: 400 BAD_REQUEST | Error Message: Contact number is already in use by another user,
	Failure in scenario: email as duplicate | Expected status code: 409 but got: 500 | Error Status: 500,
	Failure in scenario: zipcode as empty | Expected status code: 400 but got: 200,
	Failure in scenario: contentType | Expected status code: 415 but got: 200">
      <![CDATA[java.lang.AssertionError: The following asserts failed:
Failure in scenario: valid mandatory | Expected status code: 200 but got: 400 | Error Status: 400 BAD_REQUEST | Error Message: Plot number should contain alphaNumeric values only,
Failure in scenario: invalid endpoint | Expected status code: 404 but got: 400 | Error Status: 400 BAD_REQUEST | Error Message: Contact number is already in use by another user,
Failure in scenario: null endpoint | Expected status code: 404 but got: 400 | Error Status: 400 BAD_REQUEST | Error Message: Contact number is already in use by another user,
Failure in scenario: phoneNumber as duplicate | Expected status code: 409 but got: 400 | Error Status: 400 BAD_REQUEST | Error Message: Contact number is already in use by another user,
Failure in scenario: email as duplicate | Expected status code: 409 but got: 500 | Error Status: 500,
Failure in scenario: zipcode as empty | Expected status code: 400 but got: 200,
Failure in scenario: contentType | Expected status code: 415 but got: 200
at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:46)
at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:30)
at stepdefinition.updateUsertest.tearDown(updateUsertest.java:305)
]]>
    </failure>
  </testcase> <!-- runScenario -->
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="0.526"/>
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="0.743"/>
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="0.693"/>
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="0.514">
    <failure type="java.lang.AssertionError" message="The following asserts failed:
	Failure in scenario: firstName in integer | Expected status code: 400 but got: 404">
      <![CDATA[java.lang.AssertionError: The following asserts failed:
Failure in scenario: firstName in integer | Expected status code: 400 but got: 404
at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:46)
at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:30)
at stepdefinition.getUserByFirstName.validate_user_and_user_validations_for_get_request_as(getUserByFirstName.java:164)
at ✽.Validate "400" and user validations for get request as "firstName in integer"(file:///Users/mathumathibalakrishnan/git/userAPI_RestAssured/userAPI_RestAssured/src/test/resources/features/06_getUserByFirstName.feature:17)
]]>
    </failure>
  </testcase> <!-- runScenario -->
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="0.574">
    <failure type="java.lang.AssertionError" message="The following asserts failed:
	Failure in scenario: empty firstName | Expected status code: 400 but got: 404">
      <![CDATA[java.lang.AssertionError: The following asserts failed:
Failure in scenario: empty firstName | Expected status code: 400 but got: 404
at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:46)
at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:30)
at stepdefinition.getUserByFirstName.validate_user_and_user_validations_for_get_request_as(getUserByFirstName.java:164)
at ✽.Validate "400" and user validations for get request as "empty firstName"(file:///Users/mathumathibalakrishnan/git/userAPI_RestAssured/userAPI_RestAssured/src/test/resources/features/06_getUserByFirstName.feature:17)
]]>
    </failure>
  </testcase> <!-- runScenario -->
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="0.173"/>
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="0.543"/>
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="0.525"/>
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="0.536"/>
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="0.504"/>
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="0.542"/>
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="0.559"/>
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="0.504"/>
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="0.517"/>
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="0.655">
    <failure type="java.lang.AssertionError" message="The following asserts failed:
	Failure in scenario: empty userid | Expected status code: 400 but got: 404">
      <![CDATA[java.lang.AssertionError: The following asserts failed:
Failure in scenario: empty userid | Expected status code: 400 but got: 404
at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:46)
at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:30)
at stepdefinition.deletUsertest.validate_and_other_data_validations(deletUsertest.java:168)
at ✽.Validate "400" and other data validations for scenario name as "empty userid"(file:///Users/mathumathibalakrishnan/git/userAPI_RestAssured/userAPI_RestAssured/src/test/resources/features/07_deleteUserByID.feature:17)
]]>
    </failure>
  </testcase> <!-- runScenario -->
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="0.172"/>
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="0.518"/>
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="0.508"/>
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="0.518"/>
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="0.554"/>
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="0.307"/>
</testsuite> <!-- Default test -->
